docfx_yaml/utils.py

```diff
@@ -21,6 +21,7 @@ def slugify(value):
 
 
 def transform_string(app, string):
+    "*unused*"
```

Comment:
This is used in module summary extraction.

Reply:
removed